e2e/deep-matcher.ts

```diff
@@ -34,3 +34,15 @@ export function deepDataMatcher(data: any, pattern: any, path = 'ROOT'): string[
     }
     return errors;
 }
+
+export function isValidEthereumAddress(str: string) {
+    return str && str.startsWith('0x') && str.length == '0x5cd0D270C30EDa5ADa6b45a5289AFF1D425759b3'.length;
+}
+
+export function isValidImageVersion(str: string) {
```

Comment:
we actually have regular expressions in the architecture docs for matching semvers

Reply:
I know. I didn't use it explicitly.

I don't like running a function in the system and then using the same exact function in the test to check it. It's too circular. Since the regexp has unit tests, in the e2e I'm not looking to test it's inner workings but just the general idea that I got a version back. And I prefer it to be separate completely so an integration bug in the first will not influence the second.